oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDB.java

```diff
@@ -8,6 +8,26 @@
   public static final long GEOM_PRECISION_TO_LONG = 10000000L;
   public static final double GEOM_PRECISION = 1.0 / GEOM_PRECISION_TO_LONG;
 
+  /**
+   * Convert a double precision coordinate to fix precision long value
+   * 
+   * @param coordinate
+   * @return converted coordinate
+   */
+  public static long coordinateToLong(double coordinate) {
+    return (long) (coordinate * GEOM_PRECISION_TO_LONG);
+  }
+
+  /**
+   * Convert a fix precision long coordinate to a double value
```

Comment:
see above, but maybe something like this would be more readable:

```suggestion
   * Returns the double precision coordinate of a fixed precision long value.
```